src/buildercore/concurrency.py

```diff
@@ -19,4 +21,4 @@ def concurrency_for(stackname, concurrency_name):
     if concurrency_name is None:
         return 'parallel'
 
-    raise ValueError("Concurrency %s is not supported" % concurrency_name)
+    raise ValueError("Concurrency %s is not supported. Supported models: %s" % concurrency_name, concurrency_names)
```

Comment:
Needs parenthesis surrounding string formatting arguments.

Reply:
Triggered the error and confirmed it, will fix